docs/admin/access.rst

```diff
@@ -192,12 +192,12 @@ Managing per project access control
 
     This feature only works for ACL controlled projects, see :ref:`acl`.
 
-Users with :guilabel:`Can manage ACL rules for a project` privilege (see
+Users with the :guilabel:`Can manage ACL rules for a project` privilege (see
 :ref:`privileges`) can also manage users in projects with access control
-enabled on the project page. You can add or remove users to the project or make
-them owners.
+turned on through the project page. You can add users, or remove them from a projec,t or make
```

Comment:
What does access control turned on through the project page mean?
It is turned on for that, or there?

Reply:
It used to be just toggle in some older releases, now it's the Public/Protected/Private choice.